test_cases/encoding.json

```diff
@@ -44,6 +44,48 @@
           }
         ]
       }
+    },
+    {
+      "id": 3,
+      "status": "pass",
+      "endpoint": "search",
+      "description": "the Chambéry accent issue, with the accented e represented as 1 character",
```

Comment:
Adding a link to the original issue (https://github.com/pelias/api/issues/600) would be great here and in the next test